flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/OperatorChain.java

```diff
@@ -273,7 +276,7 @@ public void releaseOutputs() {
 
 		// now create the operator and give it the output collector to write its output to
 		OneInputStreamOperator<IN, OUT> chainedOperator = operatorConfig.getStreamOperator(userCodeClassloader);
-		chainedOperator.setup(containingTask, operatorConfig, output);
+		chainedOperator.setup(containingTask, operatorConfig, output, streamOutputs.size() == 0);
```

Comment:
We would not need to set this as a value in the `StreamConfig` class, because its basically a check if `StreamConfig.getOutEdgesInOrder().size == 0`. The problem is that `getOutEdgesInOrder()` expects the user code classloader. The classloader is usually not available anymore in the operators.